pkg/cmd/install/install.go

```diff
@@ -2,73 +2,23 @@ package install
 
 import (
 	"fmt"
-	"html/template"
-	"os"
-	"strings"
 
 	"github.com/spf13/cobra"
-	"github.com/thomas-armena/scrman/pkg/dir"
-	"github.com/thomas-armena/scrman/pkg/templates"
+	"github.com/thomas-armena/scrman/pkg/storage"
 )
 
-type InstallScript struct {
-	ScriptName string
-}
-
 func NewCmdInstall() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "install",
 		Short: "Install an existing bash script",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return install(args)
+			scriptName := args[0]
+			err := storage.Install(scriptName)
```

Comment:
Same issue as `storage.Create`. 

Maybe `storage.InstallScript(scriptName)` will be better.